src/seedu/addressbook/Main.java

```diff
@@ -105,12 +105,16 @@ private void recordResult(CommandResult result) {
      * @param command user command
      * @return result of the command
      */
+    //credit to #111, Cho Chih Tun for helping me and explaining
     private CommandResult executeCommand(Command command)  {
-        try {
-            command.setData(addressBook, lastShownList);
-            CommandResult result = command.execute();
```

Comment:
Is there a reason to not include these in the try block? What exceptions can they generate and how will they be handled?